templates/admin/thematic_community/member/community.html.twig

```diff
@@ -1,21 +1,22 @@
 {% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
 
 {% block field %}
-    <span class="label label-default">
-        {{ object.community.name }}
-    </span>
-    &nbsp;
-    {% for motivation in object.motivations %}
-        <span class="label label-default">
-            {{ ('admin.thematic_community.membership.motivations.'~motivation)|trans }}
+    <div style="max-width: 250px">
+        <span class="label" style="margin-right: 2px; background-color: {{ constant('App\\Entity\\ThematicCommunity\\ThematicCommunity::COLORS')[object.community.slug] }}">
+            {{ object.community.name }}
         </span>
-        &nbsp;
-    {% endfor %}
 
-    {% if object.expert %}
-        <br>
-        <span class="label label-danger">
-        Expert
-    </span>
-    {% endif %}
+        {% for motivation in object.motivations|filter(m => m is not empty) %}
```

Comment:
is it normal that the array motivations can have an empty element? maybe we should improve setter and not twig?

Reply:
normally, it's not. But from the import, it may be empty (that's how I found this actually)

Reply:
so we can improve the import command to not allow to pass an empty element?

Reply:
yes, will do that

Reply:
fixed